bucket/gimp.json

```diff
@@ -41,12 +41,17 @@
         "bin\\gimp-console-2.10.exe",
         [
             "bin\\gimp-console-2.10.exe",
-            "gimp"
+            "gimp-console"
         ],
         "bin\\gimptool-2.0.exe",
         [
             "bin\\gimptool-2.0.exe",
             "gimptool"
+        ],
+        "bin\\gimp-2.10.exe",
+        [
+            "bin\\gimp-2.10.exe",
+            "gimp"
```

Comment:
```suggestion
```

Reply:
I'm still curious about scoop's policy on invocation of the GUI stub.  Personally, I launch UIs from command line fairly often.  e.g. `notepad myfile.txt`.  I would still strongly recommend that `gimp` launches the UI and `gimp-console` launches the console, which is how the standard install behaves.  Marking as resolved, because that's a policy that scoop needs to make and I have no investment in the decision.